skfda/representation/grid.py

```diff
@@ -96,20 +96,46 @@ class FDataGrid(FData):
 
     """
 
+    class _CoordinateIterator:
+        """Internal class to iterate through the image coordinates."""
+
+        def __init__(self, fdatagrid):
+            """Create an iterator through the image coordinates."""
+            self._fdatagrid = fdatagrid
```

Comment:
The same as above.

Reply:
Reference in FDataGrid eliminated in 10565de.